kms_log/main.tf

```diff
@@ -483,9 +483,9 @@ resource "aws_cloudwatch_log_destination_policy" "subscription" {
 # create subscription filter 
 # this filter will send the kms.log events to kinesis
 resource "aws_cloudwatch_log_subscription_filter" "kinesis" {
-  depends_on      = [null_resource.kms_log_found]
-  count = var.kmslogging_service_enabled
-  
+  depends_on = [null_resource.kms_log_found]
+  count      = var.kmslogging_service_enabled
```

Comment:
Q: is this kind of horizontal alignment configurable in `terraform fmt`? IME, doing this leads to a lot of diff noise because eventually a newer, longer left-side value is added and the whole block needs to get reformatted

Reply:
Looks like it is not configurable 😿 https://www.terraform.io/docs/commands/fmt.html

Reply:
I am not sure we want to deviate from the official style guides, as we want our preference to align with the community so we can take open source code preferably as-is.  All the noise will go away as long as we all use editors that support `terraform fmt` which is the default for VSCode and a few others